third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -3694,6 +3734,35 @@ resource "google_container_cluster" "with_recurring_maintenance_window" {
 
 }
 
+func testAccContainerCluster_withMaintenanceExclusionWindow(clusterName string, w1startTime, w1endTime, w2startTime, w2endTime string) string {
+
+	return fmt.Sprintf(`
+resource "google_container_cluster" "with_maintenance_exclusion_window" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  
+  maintenance_policy {
+	recurring_window {
+		start_time = "%s"
+		end_time = "%s"
+		recurrence = "FREQ=DAILY"
+	}
+	maintenance_exclusion {
+		exclusion_name = "batch job"
+		start_time = "%s"
+		end_time = "%s"
+	}
+	maintenance_exclusion {
+		exclusion_name = "hoilday data load"
```

Comment:
```suggestion
		exclusion_name = "holiday data load"
```

Will this field accept spaces? Usually spaces don't work within map type objects

Reply:
I see it in the Request/Response, i believe its accepting